### PR TITLE
updates to the environment and the notebooks

### PR DESCRIPTION
**Pull request by max-hassani**
- updating the environment; fixing the version of the dependencies
- updating the notebooks; `create.job`, `create.structure`, `create.table`

### CODE REVIEW

**Comment by max-hassani** [comment]
> I guess we can support `numpy=1.21.41 and `h5io=0.1.4`

If I recall correctly, they are not compatible with `pyiron_base=0.3.9`. I tried to pin the base, so that none of the recent changes causes any issue in the binder environment.

File: binder/environment.yml [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 dependencies:
-- pyiron =0.3.9
-- lammps
-- nglview
-- seaborn
-- sqsgenerator
-- phonopy
-- sphinxdft
+- pyiron =0.4.5
+- pyiron_base=0.3.9
```
Comment (jan-janssen) [inline binder/environment.yml]: I guess we can remove `pyiron` and only use `pyiron_atomistics=0.2.30` and `pyiron_base=0.4.2`.

Comment (max-hassani) [reply binder/environment.yml]: I tried the notebooks with the new releases of base and atomistics, but by running pyiron tables, I faced an error, which I reported in https://github.com/pyiron/pyiron_base/issues/559.

Comment (max-hassani) [reply binder/environment.yml]: @jan-janssen, now the latest pyiron_atomistics has replaced pyiron in the environment file. I have tested the binder link, the solution notebooks work fine as well. If you agree, I can merge this.

**Review by jan-janssen — APPROVED** [review]
Looks good to me

**Pull request merged**